src/cli/cli.cpp

```diff
@@ -5760,6 +5851,25 @@ template <> otError Interpreter::Process<Cmd("parent")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli parentpriority (get,set)
+ * @code
+ * parentpriority
+ * 1
+ * Done
+ * @endcode
+ * @code
+ * parentpriority 1
+ * Done
+ * @endcode
+ * @cparam parentpriority @ca{parentpriority}
+ * @par
+ * Get the assigned parent priority value, -2 means not assigned.
+ * @par
+ * Set the assigned parent priority value: 1, 0, -1 or -2.
```

Comment:
For the description of this one, can you combine into a single statement, with the values mentioned separately.  Like we do for "bbr jitter (get,set)"